gslb/ingestion/gslb.go

```diff
@@ -677,7 +677,7 @@ func Initialize() {
 		// No need to save the Pod metadata, if running AMKO locally.
 		pod, err := kubeClient.CoreV1().Pods(gslbutils.AVISystem).Get(context.TODO(), os.Getenv("POD_NAME"), metav1.GetOptions{})
 		if err != nil {
-			gslbutils.LogAndPanic("error getting AMKO pod details: " + err.Error())
+			gslbutils.Warnf("Error getting AMKO pod details, %s.", err.Error())
 		}
 		amkoControlConfig.SaveAMKOPodObjectMeta(pod.DeepCopy())
```

Comment:
If we don't panic in 680, won't we crash here? Assuming pod is nil, `pod.DeepCopy()` will cause a runtime panic?

Reply:
you're right, fixed. thanks